views/table_chart_plugin_library_selector.inc

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Contains table_chart_plugin_library_selector.
+ */
+
+class table_chart_plugin_library_selector extends views_plugin_display_extender {
+  // this is a testviews_plugin_display_extender x
+  function options_definition_alter(&$options) {
+      $options['chartlib_selector'] = array('default' => array());
+  }
+  
+  function options_summary(&$categories, &$options) {
+    $chartlib_selector = $this->display->get_option('chartlib_selector');
+    $options['chartlib_selector'] = array(
+      'category' => 'other',
+      'title' => t('Chart Library Selector'),
+      'description' => t('Add data chartlib to your display wrapper and fields.'),
+      'value' => !empty($chartlib_selector) ? t('chartlib added'): t('None'),
```

Comment:
``chartlib added`` is that supposed to be a machine value? Or text shown to the user. If it's text it should be more properly formatted grammatically. If not then it shouldn't be translated and contain no spaces.